### PR TITLE
LPS-177384 Remove Stream usages for module: commerce-test

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from #3744

### CODE REVIEW

**Comment by catarineaguiar** [comment]
@julschong formatting issue:

```
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/order/engine/test/CommerceOrderEngineTest.java expected:<...mmerceOrderStatus ->[
     [java] 					]commerceOrderStatus....> but was:<...mmerceOrderStatus ->[ ]commerceOrderStatus....>
     [java] 
     [java] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:444)
     [java] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:286)
```
full log [here](https://testray.liferay.com/reports/production/logs/2023-03/test-1-40/test-portal-source-format/394/source-format-current-branch-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

After fixing it, run commerce suite please. Thanks!

**Comment by julschong** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/order/engine/test/CommerceOrderEngineTest.java [hunk 1]
```diff
@@ -1005,19 +999,12 @@ public void testGetNextOrderStatusesWhileOrderNotOpen() throws Exception {
...
-				Collectors.toList()
-			);
-
-		Assert.assertFalse(inProgressCommerceOrderStatuses.isEmpty());
+		Assert.assertTrue(
+			ListUtil.exists(
+				_commerceOrderEngine.getNextCommerceOrderStatuses(
+					_commerceOrder),
+				commerceOrderStatus ->
+					commerceOrderStatus.getPriority() == -1));
```
Comment (julschong) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/order/engine/test/CommerceOrderEngineTest.java:1007]: Changed here:
assert that at least 1 CommerceOrderStatus satisfy entry.getPriority() == -1

**Comment by julschong** [comment]
Will change to only using for loop for more simplicity

**Comment by dantewang** [comment]
ci:forward:force

**Pull request closed without merging**